.gitignore

```diff
@@ -60,6 +60,7 @@ extra/dist/tarantool.logrotate
 extra/dist/tarantool@.service
```

Comment:
In the commit message to this commit you said the `mp_*` symbols are used only by the tests. Is there a ticket to hide them and make the tests use their own msgpuck build or just inline functions? If not, then it must either be fixed here, or the ticket must be created so as not to forget about the problem.

Reply:
I will do it in a separate PR for this task. The target of this patch is to unblock the teamC and it must be in the release, further patches for this ticket may be pushed after release.